### ISSUE TITLE
Can't add parameters for DecimalField used in ArrayField (peewee 2.10.2)

### ISSUE THREAD

**Issue by ssrebelious** [coleifer/peewee#1608]
I have a model  where I need to define custom parameters for DecimalField that is used inside ArrayField:
```python
class Pricing(Model):
    base_cost = ArrayField(
        DecimalField,
        max_digits=16,
        decimal_places=2,
        help_text='Base cost',
        null=False
    )
    class Meta:
        database = postgres_db
```
This results in error telling me that `max_digits` and `decimal_places` are not supported parameters for `ArrayField`. I took a quick peek into `ArrayField` class and I guess that the `__init__` method might be an issue: *args and **kwargs are passed as parameters to `self.__field = field_class(*args, **kwargs)` and `super(ArrayField, self).__init__(*args, **kwargs)`.

As a workaround I could subclass `DecimalFiled` with required parameters and pass it to `ArrayField`. Is there another way?

**Comment by coleifer** [comment]
That's a nice workaround but I'll see what I can do to offer better support.

**Comment by coleifer** [comment]
This fix will be in the 3.4.0 release. If you plan to continue using 2.10 then the subclassing solution is totally fine.
